README.md

```diff
@@ -227,22 +227,22 @@ bionemo-esm2-recipe \
 
```

Comment:
--dest my_config.yaml, @skothenhill-nv , I would suggest to search for all occurrences of .json in the codebase

Reply:
Imho it is a bit unclear what those recipes include

Ie 

> Alternatively, we provide a validated and serialized configuration file entrypoint for executing the same workflow. Recipes
> are available for 8m, 650m, and 3b ESM2 models. You may select which preset config to use by setting the `--recipe` parameter.

Does it mean that each config includes multiple recipes? From the command line is seems so. What would happen is --recipe is not provided? All the recipes in the config are launched?